src/components/SmallShareButtons/SmallShareButtons.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  ButtonsWrapper,
+  ShareButtonContainer,
+} from './SmallShareButtons.style';
+import {
+  FacebookShareButtonInner,
+  TwitterShareButtonInner,
+} from 'components/ShareButtons';
+import { COLOR_MAP } from 'common/colors';
+import LinkIcon from '@material-ui/icons/Link';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+//TODO (Chelsi): Add in final share quote
+const SmallShareButtons = (props: { shareUrl: string }) => {
+  const url = props.shareUrl;
+
+  const socialProps = {
+    url,
+    quote: '',
```

Comment:
We probably need the quote to be a prop for this component, as we will likely use it in other features